schema/Comment.schema.yaml

```diff
@@ -0,0 +1,23 @@
+title: Comment
+'@id': schema:Comment
+extends: CreativeWork
+role: primary
+status: stable
+category: metadata
+description: A comment on an item, e.g on a Article, or SoftwareSourceCode.
+$comment: |
+  Use the `about` property to define the item that a comment is on.
+  The `content` property should be used for the structured content of the
+  Comment, in preference to the schema.org `text` property which is is expected to
+  have the plain text content.
+properties:
```

Comment:
I'm wondering about the design choice to have both a parent id and a list of replies (Other comments) to the data structure - is the idea that both could be optional, so you could have:

 - a top level comment without a parent
 - not using any parent id, and just nesting comments as replies
 - only using a parent id, and having the implementation crawl upwards to derive the history (parents)
 - an implementation that both implements having the parentId and a list of comments in replies?

Reply:
Yes, I wonder about that too. For our current use case (where we are fetching comments from Google Docs and GitHub PRs)  I suspect that we will only use "not using any parent id, and just nesting comments as replies" model.

So I mainly included `parentItem` because that is in schema.org. However, I can see it's usefulness for those other scenarios. For example, when getting an individual comment, GitHub's API has a `in_reply_to_id` field https://developer.github.com/v3/pulls/comments/#response-2

Reply:
okay, that makes sense. The design allows for flexibility of the implementation to do what works best.